Integrations/integration-UrlScan.yml

```diff
@@ -0,0 +1,411 @@
+description: Urlscan.io reputation
+detaileddescription: |-
+  This integration checks domain information from the urlscan.io Database.
+  No authentication is needed.
+  If you want to submit a scan you need an API key.
+  The API key provided by contacting the urlscan.io directly  by email.
+configuration:
+- display: URL for Urlscan API
+  name: url
+  defaultvalue: https://urlscan.io/api/v1/
+  type: 0
+  required: false
+- display: API Key (needed only for submitting URLs for scanning)
+  name: apikey
+  defaultvalue: ""
+  type: 0
+  required: false
+- display: Use system proxy settings
+  name: proxy
+  defaultvalue: "false"
+  type: 8
+  required: false
+script:
+  script: |
+    var sendRequest = function(method, arg, body, headers, img) {
+
+        url = params.url;
+        if (url[url.length - 1] === '/') {
```

Comment:
won't handle with // at the end. change to the regex in sandblast or wildfire, or use while